Amazon.SQS/src/Extensions.SQS/Collector/SqsQueueAsyncCollector.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Amazon;
+using Amazon.Runtime;
+using Amazon.SQS;
+using Azure.Functions.Extensions.SQS.Commons;
+using Microsoft.Azure.WebJobs;
+
+namespace Azure.Functions.Extensions.SQS.Collector
+{
+	public class SqsQueueAsyncCollector : IAsyncCollector<SqsQueueMessage>
+	{
+		private AmazonSQSClient AmazonSQSClient { get; }
+		private SqsQueueTriggerAttribute TriggerParameters { get; }
```

Comment:
There is just one caveat here. This would limit use of the output bindings to only those functions that are using queue as a trigger as well. It is very well possible for a function to get triggered by some other mechanism but want to queue messages for another consumer.
That would be a different work item though.